module/spl/spl-vnode.c

```diff
@@ -641,64 +641,6 @@ vn_areleasef(int fd, uf_info_t *fip)
 } /* releasef() */
 EXPORT_SYMBOL(areleasef);
 
-
-static void
-vn_set_fs_pwd(struct fs_struct *fs, struct path *path)
-{
-	struct path old_pwd;
-
-#ifdef HAVE_FS_STRUCT_SPINLOCK
```

Comment:
You should now be able to remove `config/kernel-spinlock.m4` too.

Reply:
Updated.